app/grants/templates/grants/invoice.html

```diff
@@ -0,0 +1,212 @@
+    }
+
+    .invoice-box table tr.total td:nth-child(2) {
+      border-top: 2px solid #eee;
+      font-weight: bold;
+    }
+
+    @media only screen and (max-width: 600px) {
+      .invoice-box table tr.top table td {
+        width: 100%;
+        display: block;
+        text-align: center;
+      }
+
+      .invoice-box table tr.information table td {
+        width: 100%;
+        display: block;
+        text-align: center;
+      }
+    }
+
+    /** RTL **/
+    .rtl {
+      direction: rtl
+    }
+
+    .rtl table {
+      text-align: right;
+    }
+
+    .rtl table tr td:nth-child(2) {
+      text-align: left;
+    }
+    </style>
+  </head>
+
+  <body class="g-font-muli">
+    <div class="invoice-box">
+      {% if grant.network != 'mainnet' %}
+        <div style="color:red; border: 1px red solid; padding: 5px;">
+          {% trans "Note: This is not a mainnet grant, it is a " %} {{grant.network}} {% trans "grant, and this invoice is only for demonstration purposes." %}
+        </div>
+      {% endif %}
+      <table cellpadding="0" cellspacing="0">
+        <tr class="top">
+          <td colspan="2">
+            <table>
+              <tr>
+                <td class="title">
+                  <img src="{% static "v2/images/presskit/GitcoinLogoText.svg" %}" style="width:100%; max-width:300px;">
+                </td>
+
+                <td>
+                  {% trans "Invoice" %} #: {{contribution.pk}}<br>
+                  {% trans "Created" %}: {{contribution.created_on}}<br>
+                  {% trans "Status" %}: {{subscription.status}}<br>
+                  {% trans "For" %}: <a href="{{subscription.grant.url}}">{% trans "Gitcoin Grant" %} {{subscription.grant.pk}}</a> <br>
+                </td>
+              </tr>
+            </table>
+          </td>
+        </tr>
+
+        <tr class="information">
+          <td colspan="2">
+            <table>
+              <tr>
+                <td>
+                  {% trans "Prepared by" %} <br>
+                  Gitcoin Core LLC<br>
+                  ConsenSys Inc. <br>
+                  49 Bogart Street Suite 22 <br>
+                  Brooklyn, NY 11206 <br>
+                </td>
+
+                <td>
+                  {% trans "Prepared for Funder" %}: <br>
+                  {{subscription.grant.admin_address}}<br>
+                  @{{subscription.grant.admin_profile.handle}}<br>
+                  {{subscription.grant.admin_profile.email}}<br>
+                </td>
+              </tr>
+            </table>
+          </td>
+        </tr>
+
+        <tr class="heading">
+          <td>
+            {% trans "Payment Method" %}
+          </td>
+
+          <td>
+            {% trans "Account" %} #
+          </td>
+        </tr>
+
+        <tr class="details">
+          <td>
+            {% trans "Directly to counterparties via Ethereum Blockchain" %}
+          </td>
+
+          <td>
+            0x2af47a65da8cd66729b4209c22017d6a5c2d2400
```

Comment:
Whats up with this hardcoded value?

Reply:
I based this invoice on the invoice template for bounties, I took the value from there. As I understand this address represents the gitcoin address to manage such transactions. This value is hardcoded in other places in the codebase, what is recommended in this cases? @danlipert

Reply:
So I took a look and actually its the Bounties Network v1 Contract Address, which doesn't make sense here... lets use the grant contract address for each grant instead.